crates/viewer/re_view_spatial/tests/draw_order.rs

```diff
@@ -0,0 +1,226 @@
+
+            builder.with_archetype(
+                RowId::new(),
+                TimePoint::default(),
+                &re_types::archetypes::Image::from_color_model_and_tensor(
+                    re_types::datatypes::ColorModel::RGB,
+                    image,
+                )
+                .unwrap()
+                .with_draw_order(1.1),
+            )
+        });
+
+        test_context.log_entity("2d_layering/lines_behind_rect".into(), |builder| {
+            builder.with_archetype(
+                RowId::new(),
+                TimePoint::default(),
+                &re_types::archetypes::LineStrips2D::new(std::iter::once(
+                    (0..20)
+                        .map(|i| i as f32)
+                        .map(|i| (i * 20.0, i % 2.0 * 100.0 + 70.0)),
+                ))
+                .with_draw_order(1.25)
+                .with_colors([0xFF0000FF]),
+            )
+        });
+
+        test_context.log_entity(
+            "2d_layering/rect_between_top_and_middle".into(),
+            |builder| {
+                builder.with_archetype(
+                    RowId::new(),
+                    TimePoint::default(),
+                    &re_types::archetypes::Boxes2D::from_mins_and_sizes(
+                        [(64.0, 32.0)],
+                        [(256.0, 128.0)],
+                    )
+                    .with_draw_order(1.5)
+                    .with_colors([0x000000FF]),
+                )
+            },
+        );
+
+        test_context.log_entity(
+            "2d_layering/points_between_top_and_middle".into(),
+            |builder| {
+                builder.with_archetype(
+                    RowId::new(),
+                    TimePoint::default(),
+                    &re_types::archetypes::Points2D::new((0..16 * 16).map(|i| i as f32).map(|i| {
+                        (
+                            32.0 + (i as i32 / 16) as f32 * 16.0,
+                            32.0 + (i as i32 % 16) as f32 * 16.0,
+                        )
+                    }))
+                    .with_draw_order(1.51),
+                )
+            },
+        );
+
+        // Small white square on top
+        test_context.log_entity("2d_layering/top".into(), |builder| {
+            let mut image = Array::<u8, _>::zeros((128, 128, 3).f());
+            image.fill(255);
+
+            builder.with_archetype(
+                RowId::new(),
+                TimePoint::default(),
+                &re_types::archetypes::Image::from_color_model_and_tensor(
+                    re_types::datatypes::ColorModel::RGB,
+                    image,
+                )
+                .unwrap()
+                .with_draw_order(2.0),
+            )
+        });
+    }
+
+    let view_id = setup_blueprint(&mut test_context);
+    run_view_ui_and_save_snapshot(
+        &mut test_context,
+        view_id,
+        "draw_order",
+        egui::vec2(300.0, 150.0) * 2.0,
```

Comment:
curious resolution

Reply:
The one I copy pasted felt too small for this... in fact it felt 2x too small :x